okhttp-urlconnection/pom.xml

```diff
@@ -61,6 +61,17 @@
           </links>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Automatic-Module-Name>${project.groupId}.urlconnection</Automatic-Module-Name>
```

Comment:
I’m trying to find out whether we can have two modules, where one is OkHttp and the other is this one, and their subpackages overlap